guides/source/_welcome.html.erb

```diff
@@ -16,9 +16,9 @@
 <% end %>
 <p>
 The guides for earlier releases:
-<a href="http://guides.rubyonrails.org/v4.2.0/">Rails 4.2.0</a>,
-<a href="http://guides.rubyonrails.org/v4.1.8/">Rails 4.1.8</a>,
-<a href="http://guides.rubyonrails.org/v4.0.12/">Rails 4.0.12</a>,
-<a href="http://guides.rubyonrails.org/v3.2.21/">Rails 3.2.21</a> and
+<a href="http://guides.rubyonrails.org/v4.2.4/">Rails 4.2.4</a>,
```

Comment:
@fxn do you think we should create aliases with just the major and minor version? I don't think it is a good idea to keep updating this every release.